packages/fuel-indexer-tests/src/fixtures.rs

```diff
@@ -287,7 +287,7 @@ pub async fn api_server_app_postgres(
         verbose: true,
         local_fuel_node: false,
         indexer_net_config: false,
-        fuel_node: FuelNodeConfig::default(),
+        fuel_node: FuelClientConfig::default(),
```

Comment:
Can we also rename `fuel_node` to `fuel_client`?

Reply:
[See comment](https://github.com/FuelLabs/fuel-indexer/pull/1007#discussion_r1238597585)